x-pack/legacy/plugins/siem/public/components/ml/api/throw_if_not_ok.ts

```diff
@@ -83,7 +71,7 @@ export const throwIfErrorAttachedToSetup = (setupResponse: SetupMlResponse): voi
 
   const dataFeedErrors = setupResponse.datafeeds.reduce<string[]>(
     (accum, dataFeed) =>
-      dataFeed.error != null
+      dataFeed.error != null && jobIdErrorFilter.includes(`datafeed-${dataFeed.id}`)
```

Comment:
I traced this up but did not see any `datafeed-` prefixes being adde to `jobIdErrorFilter` so if they're not being passed down they are all being filtered out at the moment?

Reply:
Ahh, yes! Good catch! This is actually the inverse though. So the format of `datafeed.id` is always `datafeed-${job.id}`, so we'll want to extract the `jobId` from the `datafeed.id` to see if it's in the whitelist.

e.g. 
``` ts
jobIdErrorFilter.includes(dataFeed.id.substr('datafeed-'.length))
```